.github/ISSUE_TEMPLATE/diagnostics.yaml

```diff
@@ -1,6 +1,6 @@
 name: Diagnostic issue
-description: Create a bug report or feature request for a change to `rustc`'s error output
-labels: ["A-diagnostics", "T-compiler"]
+description: Create a bug report or feature request for a change to `rustc`' or `rustdoc`'s error output or lints
```

Comment:
Lint issues should generally have A-lints tho, not A-diagnostics (there are some specific exceptions)